### PR TITLE
Handle nulls properly in the contract zone importer

### PR DESCRIPTION
**Pull request by tuomas777**
Before this fix, nulls in the source data ended up in our db as `"None"` instead of the correct empty value `""`.

No tests 😞 Adding one for this would have been a bit too much in this situation where there is no existing boilerplate for importer tests. Hopefully importer tests will be implemented for this project someday. But tested manually and works like a charm!

Refs [PS-144](https://helsinkisolutionoffice.atlassian.net/browse/PS-144)

[PS-144]: https://helsinkisolutionoffice.atlassian.net/browse/PS-144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**PUISTOTALKOOT-API branch is deployed to platta: https://puistotalkoot-pr153.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by danipran — APPROVED** [review]

File: areas/importer/helsinki.py [hunk 1]
```diff
@@ -102,10 +102,8 @@ def _process_contract_zones(self, data_source, force=False):
 
     @staticmethod
     def _get_attribute_safe(feature, attribute):
-        try:
-            return str(feature[attribute])
```
[the remark this answers is not in the document]
Comment (tuomas777) [reply areas/importer/helsinki.py:106]: Yes very good point 🦜 In addition to the values not being stripped, the new implementation didn't even work with missing fields as it should have 😅

**Comment by tuomas777** [comment]
Pushed take two.

**Comment by terovirtanen** [comment]
**PUISTOTALKOOT-API branch is deployed to platta: https://puistotalkoot-pr153.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by voneiden — APPROVED** [review]
:cowboy_hat_face:

**Pull request merged**